src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -25,22 +26,28 @@
             return new Parcel[] {
                 new Parcel(new TrackingNumber("RR999999999SG"), new Name("Alex Yeoh"), new Phone("87438807"),
                         new Email("alexyeoh@example.com"), new Address("Blk 29 Lor 30 Geylang, #06-40 s398362"),
-                        new DeliveryDate("01-01-2001"), getTagSet("friends")),
+                        new DeliveryDate("01-01-2001"), Status.getStatusInstance("PENDING"),
+                        getTagSet("friends")),
                 new Parcel(new TrackingNumber("RR111111111SG"), new Name("Bernice Yu"), new Phone("99272758"),
                         new Email("berniceyu@example.com"), new Address("Blk 326 Serangoon Ave 3, #07-18 S550326"),
-                        new DeliveryDate("02-02-2002"), getTagSet("colleagues", "friends")),
+                        new DeliveryDate("02-02-2002"), Status.getStatusInstance("pending"),
```

Comment:
Hey Kennard, are you going to standardize the type case of Pending (like "PENDING" or "pending)

Reply:
case insensitive XD